dataactvalidator/filestreaming/csvS3Reader.py

```diff
@@ -34,11 +34,6 @@ def openFile(self, region, bucket, filename, csvSchema, bucketName, errorFilenam
         super(CsvS3Reader, self).openFile(
             region, bucket, filename, csvSchema, bucketName, errorFilename)
 
-    def downloadFile(self, region, bucket, filename, targetLocation):
-        """ After opening a file, download entire file to filename """
-        self.s3File = initializeFile(region, bucket, filename)
-        self.s3File.get_contents_to_filename(targetLocation)
-
```

Comment:
:+1: the function name doesn't appear anywhere else, so I think we're safe unless we're doing something super dynamic.